packages/clay-css/src/scss/_components.scss

```diff
@@ -33,6 +33,8 @@
 @import "components/_form-validation";
 @import "components/_icons";
 @import "components/_input-groups";
+@import "components/_input-move-boxes";
+@import "components/_select-box";
```

Comment:
Don't need `@import "components/_input-move-boxes";` and `@import "components/_select-box";`